packages/cozy-pouch-link/src/PouchManager.js

```diff
@@ -351,6 +351,45 @@ class PouchManager {
     this.warmedUpQueries = {}
     await this.storage.destroyWarmedUpQueries()
   }
+
+  /**
+   * Adds a new doctype to the list of managed doctypes, sets its replication options,
+   * creates a new PouchDB instance for it, and sets up the query engine.
+   *
+   * @param {string} doctype - The name of the doctype to add.
+   * @param {Object} options - The replication options for the doctype.
+   */
+  addDoctype(doctype, options) {
```

Comment:
Maybe this should be called from the `init()` in PouchManager ? To avoid code duplication and forget to change this code if anything change in the future